spec/factories/teams.rb

```diff
@@ -1,6 +1,6 @@
 FactoryGirl.define do
   factory :team do
     sequence(:name) { |n| "Patriots#{n}" }
-    city "New England"
+    location "New England"
```

Comment:
Yeah I think the "correct" way might be have this literally in its own PR. But in reality, that's very silly and this is fine. How did you catch this by the way? Failing tests?